### ISSUE TITLE
TPF Tools, Autofix ignores duplicates

### ISSUE THREAD

**Issue by ghost** [ME3Explorer/ME3Explorer#317]
@KFreon -- This should NOT be fixed for the stable.

Relocated from a different issue.

![autofixdetectfail](https://cloud.githubusercontent.com/assets/13645804/14532005/4bc4f9e8-0214-11e6-8d8b-30d2cdc6ade5.PNG)

What's happening here seems to be a result of having duplicate textures that _don't_ need fixing. If I load up a single texture that needs fixing, it fixes fine:

![needsaf](https://cloud.githubusercontent.com/assets/13645804/14532065/8aeb342a-0214-11e6-8432-067fd69e63d9.PNG)
![af_success](https://cloud.githubusercontent.com/assets/13645804/14532071/8fd4baa6-0214-11e6-8576-ed5753c4de41.PNG)

If I go back to the first 4 textures I used and _only_ load the ones that need fixing, autofix only fixes the first one and leaves the second one borked:

![autofix weird](https://cloud.githubusercontent.com/assets/13645804/14532165/f82007d2-0214-11e6-9725-ce4f9ec5be58.PNG)

To summarize:
- If the first duplicate hash texture autofix encounters doesn't need fixing, it will not autofix any other duplicates -- _even if they need to be fixed_
- If the first duplicate hash texture autofix encounters does need fixing, it will fix that one and not any others -- _even if they need to be fixed_

Ideally, autofix will fix all textures regardless of whether they are hash duplicates. That way the user can promote after autofix and not just before.
